presto-raptor/src/main/java/com/facebook/presto/raptor/RaptorColumnIdentity.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.raptor;
+
+import com.facebook.presto.spi.ColumnIdentity;
+
+import java.util.Objects;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static com.google.common.primitives.Longs.toByteArray;
+
+public class RaptorColumnIdentity
+        implements ColumnIdentity
+{
+    private final long columnId;
+
+    public RaptorColumnIdentity(long columnId)
+    {
+        this.columnId = columnId;
+    }
+
+    @Override
+    public byte[] serialize()
+    {
+        return toByteArray(columnId);
```

Comment:
To assure that serialized format is backwards compatible across releases, there should be a test with hard coded serialized data.